styles/globals.css

```diff
@@ -88,6 +88,10 @@ input[type="search"]::-webkit-search-cancel-button {
   z-index: 50;
 }
 
+div[role="dialog"] {
+  scrollbar-color: #c2c9d2 transparent;
+}
+
```

Comment:
Refactor (blocking): This doesn't fix the issue, but debugging it a bit revealed the actual issue.

```suggestion
```

The highlight input form has the CSS class `overflow-y-scroll` set on the dialog. If you set it to `overflow-y-auto`, we see all the rounded corners in Firefox.

https://github.com/maharjanraj/open-souced-app/blob/f7f1163ce98a9a638525095cebb5bbe6cf3418a4/components/molecules/HighlightInput/highlight-input-form.tsx#L537

This won't fix the issue, but we could move the `overflow-y-scroll` to an inner element instead of the main dialog element. That way we preserve the rounded corners and we can still scroll the dialog content if necessary.

Reply:
@nickytonline thanks for the review and change request. I tried moving the overflow-y-scroll to inner element, however the top of the scrollbar now overlaps with the close icon.
<img width="720" alt="Screenshot 2023-11-21 at 12 51 03 PM" src="https://github.com/open-sauced/app/assets/12781253/b023f122-8106-4d2f-9aa6-f685bb314c36">

Is it okay to add some margin on the top of the DiagLogHeader so that scrollbar begins below the close icon level?
After adding margin:
<img width="711" alt="Screenshot 2023-11-21 at 1 07 33 PM" src="https://github.com/open-sauced/app/assets/12781253/6dc90a2a-9d09-49a8-b788-54cd1ee59ae5">

Reply:
New update: I have made only the form component scrollable so now the header is fixed.
<img width="736" alt="image" src="https://github.com/open-sauced/app/assets/12781253/ed40c7d5-2f36-4b2b-bcb6-e4804582c4bc">
CC: @nickytonline, @jpmcb, @brandonroberts